resume/routes.py

```diff
@@ -11,6 +11,8 @@
 load_dotenv()
 
 title = "Posts"
+# Setting the correct path for configuration to wkhtmltopdf.exe
+path_to_wkhtmltopdf = r'C:\Program Files\wkhtmltopdf\bin\wkhtmltopdf.exe'
```

Comment:
I think this path would not work for linux users is there any way this is supported for all Os

Reply:
Rest seems good , can you send the PR in the development branch from where I will merge to master , thats the flow I am adopting

Reply:
> I think this path would not work for linux users is there any way this can be supported for all Os.
> Is there a reason to add the path , what does it accomplish?

@DiptoChakrabarty when users download wkhtmltopdf, the configuration needs to be set up for the conversion to work. The ```pdfkit.from_string()``` function needs to be able to access the ```.exe``` application of wkhtmltopdf, else it will throw errors stating it couldn't find the PATH to the ```.exe``` file. That's why it is necessary to define the path to the ```.exe``` application and provide it as the input to the ```pdfkit.configuration()``` function, which is later used by the ```pdfkit.from_string()``` function.

Also, I searched on the internet, there's no option but to manually set the PATH up. I will still try searching for a solution, maybe I have missed.

Reply:
> Rest seems good , can you send the PR in the development branch from where I will merge to master , thats the flow I am adopting

I will send in the PR after resolving two more bugs that I came across today. Mentioned in the **[Issue thread](https://github.com/DiptoChakrabarty/Resume-Generator/issues/30#issuecomment-706620600)**